src/ert/cli/workflow.py

```diff
@@ -20,7 +20,7 @@ def execute_workflow(
         msg = "Workflow {} is not in the list of available workflows"
         logger.error(msg.format(workflow_name))
         return
-    runner = WorkflowRunner(workflow, storage, ert_config=ert_config)
+    runner = WorkflowRunner(workflow=workflow, storage=storage, ert_config=ert_config)
```

Comment:
OK change, but purely cosmetic and unrelated to this PR? Could be in a separate commit